ibraheemosule/skoolizy
# za - Updated pathname for the route and removed record grade from the side nav

# Why this PR?
Removed the pathname of the route from mobile view and updated the pathname to make sure sub routes are properly arranged.

# What does this PR do?
It shows the pathname of the current page you have navigated to only on desktop screens and ensures that the route names are displayed properly. Also the record grade component was modified, removing from the route from the primary navigation and  moving it to a sub route.

# How to test your work?
You should see that the path names are not displayed on mobile screens anymore and  you should see that sub routes are properly spaced and arranged. Also you should see that the record grade component has been removed from the primary menu and added as a sub menu.

# Pages to see your changes?
- All pages
- Classroom.tsx
